sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -555,6 +564,9 @@ def begin_analyze_healthcare_entities(  # type: ignore
             Union[~azure.ai.textanalytics.AnalyzeHealthcareEntitiesResult, ~azure.ai.textanalytics.DocumentError]]]
```

Comment:
I can't comment on it, but for healthcare, the `disable_service_logs` default on the service is set to `true`. not sure if you would like to change the docstring to reflect that too

Reply:
FYI, not sure if you already have it, but it is also `true` for the PII task. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesAction.cs#L53 as an example

Reply:
Got it - will update. Just to clarify - does it default to True for all endpoints or just healthcare and Pii?

Reply:
Just for healthcare and PII (both singular and PII action)